merino/curated_recommendations/corpus_backends/protocol.py

```diff
@@ -22,10 +22,19 @@ class Topic(str, Enum):
     TECH = ("tech",)
     TRAVEL = ("travel",)
 
-    @staticmethod
-    def values():
-        """Map enum values & returns"""
-        return Topic._value2member_map_
+
+@unique
+class RecommendationSurfaceId(str, Enum):
+    """Defines the possible recommendation surfaces."""
+
+    HOME = "HOME"
```

Comment:
curious what this surface is - i don't think this will return any results from the corpus...

Reply:
It's Pocket Home

Reply:
so do we remove?

Reply:
I'm on the fence. I slightly lean towards keeping this value, because it is a valid value that this enum can take, even if we won't use it in this service, for now. If anyone feels more strongly that it should be removed, I'm fine with that, too.

Reply:
Maybe the least we should do is add a comment:
```suggestion
    HOME = "HOME"  # Pocket Home
```

Reply:
Sorry, thinking about this again, we should remove it. `HOME` is not a scheduled surface id, and we should probably make a distinction between the two. We should probably rename `RecommendationSurfaceId` to `ScheduledSurfaceId` as well.